### PR TITLE
Alexander hristov

### PR DESCRIPTION
**Pull request by aleksandrhristov**
SumatorTests

### CODE REVIEW

File: src/Demo.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+    public static void main(String[] args) throws IntegersOnlyStringExceptions {
+        Sumator su = new Sumator();
+        Scanner sc = new Scanner(System.in);
+        String str1 = sc.nextLine();
+        String str2 = sc.nextLine();
+        String sum = "";
+        try {
+            sum = su.Sum(str1, str2);
```
Comment (mgenov) [inline src/Demo.java]: method names in Java should start in lower case by convention.

File: src/Test.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+public class Test  {
+    @org.junit.Test
+
+    public void correctSymbolRecieve(){
+        Sumator testone = new Sumator();
+        String result = testone.Sum("#","$");
+        assertEquals("",result);
+    }
+    @org.junit.Test(expected=IllegalArgumentException.class)
+    public void testIllegalArgumentException() {
```
Comment (mgenov) [inline src/Test.java]: method names should not start with test when juni4 is used as annotation already mark this method as test method. 

You are verifying the scenario when both arguments are null as I can see, so you test should look like: 
```java
@Test(expected=IllegalArgumnetException.class)
public void bothArgumentsAreNull() {
  Sumator sumator = new Sumator();
  sumator.sum(null, null); 
}
```

File: src/Test.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * Created by clouway on 22.06.16.
+ */
+import org.junit.*;
+import org.hamcrest.*;
+import org.junit.internal.builders.JUnit3Builder;
+import org.junit.internal.builders.JUnit4Builder;
+
+import static org.junit.Assert.*;
+
+public class Test  {
```
Comment (mgenov) [inline src/Test.java]: Name of the test should give information which behaviour is testing or at least which class.

for testing of Store class, your test should be named StoreTest. 

If you want to give better name for your test class, you can use name that describes the behaviour which is testing. 

Examples: PrintReceiveProtocolsInWarehouseTest -> you are testing all scenarios which are occurring during the receiving of products in the warehouse.

File: src/Test.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    @org.junit.Test
```
Comment (mgenov) [inline src/Test.java]: This annotation looks like some flying one, please try to format your code before you submit the PR.

Comment (aleksandrhristov) [inline src/Test.java]: fixed

File: src/Test.java (hunk 3 above)
Comment (aleksandrhristov) [inline src/Test.java]: ok thanks for the advice . Fixed

File: src/Test.java (hunk 2 above)
Comment (aleksandrhristov) [inline src/Test.java]: Fixed

File: src/Demo.java (hunk 1 above)
Comment (aleksandrhristov) [inline src/Demo.java]: Fixed now ....

**Comment by aleksandrhristov** [comment]
Updated

**Comment by aleksandrhristov** [comment]
Again Updated Code

File: src/SumatorTest.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void bothArgumentsAreNull() {
+        Sumator sumator = new Sumator();
+        sumator.sum(null, null);
+    }
+
+    @Test(expected = NumberFormatException.class)
+    public void testNumberFormatException() {
```
Comment (mgenov) [inline src/SumatorTest.java]: .

Comment (mgenov) [inline src/SumatorTest.java]: testNumberFormatException -> test the number format exceptions ? 

Are you sure that you are doing this in your test ?

**Comment by aleksandrhristov** [comment]
Updated

**Comment by mgenov** [comment]
Name of the test method was misleading and I though that you are going to fix it as you fix the other ones to. The only difference is that I didn't gave you final name of the test and I was expected that from you. 

LGTM !

**Pull request merged**